src/test/ui/stability-attribute/stability-attribute-issue-43027.rs

```diff
@@ -1,8 +1,9 @@
+// check-pass
 #![feature(staged_api)]
 #![stable(feature = "test", since = "0")]
 
 #[stable(feature = "test", since = "0")]
-pub struct Reverse<T>(pub T); //~ ERROR field has missing stability attribute
+pub struct Reverse<T>(pub T);
```

Comment:
I feel like this is potentially dangerous change that also limits our options in the future.

I can see an use-case in adding an unstable new field to a struct and not impacting any stable uses of the struct.

Similarly for `#[non_exhaustive]` enums (the only kind of enum to which it is valid to add a variant), though perhaps significantly more debatable depending on the specific case.

Reply:
> I can see an use-case in adding an unstable new field to a struct and not impacting any stable uses of the struct.

I'm not sure I follow. This PR makes the linked code semantically the same as the current 

```rust
#[stable(feature = "test", since = "0")]
pub struct Reverse<T>(#[stable(feature = "test", since = "0")] pub T);
```

but you can still add `unstable` (and `stable` with different feature name) annotations to individual fields as you add to them. You currently get a compile error with the presented code. I don't see how this change limits us?

Reply:
I guess the point is more that we might *accidentally* stabilize struct fields without intending to?

Reply:
Yeah, the reason this comment ended up being here is because I’m expressing my observation based on the removed error message, not because the struct is in any way interesting.

@nikomatsakis summarized my point well.